### PR TITLE
Update 2020 rates

### LINKED ISSUE #91: The tax brackets are wrong for 2020
It's announced that there will be 2 brackets (up to € 68,508 37.35%, from € 68,508 49.50%) for 2020. ([ref](https://www.belastingdienst.nl/wps/wcm/connect/bldcontentnl/belastingdienst/prive/inkomstenbelasting/heffingskortingen_boxen_tarieven/boxen_en_tarieven/overzicht_tarieven_en_schijven/u-hebt-in-2020-nog-niet-aow-leeftijd)) which makes the brackets that are added with this commit wrong: 1fd5304

Could you please update the tax brackets?

### PR DESCRIPTION
**Pull request by eduardomourar**
This fixes #91 and #93.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -5,13 +5,23 @@ import rulingComponent from './ruling/ruling.component';
 import partnerSectionComponent from './parner-section/partner-section.component';
 import wordCalcComponent from './world-calc/world-calc';
 
-angular.module('dit-calculator', ['ngMaterial'], ['$locationProvider', function($locationProvider){
+var App = angular.module('dit-calculator', ['ngMaterial']);
```
Comment (stevermeister) [inline src/app.js]: let's use modern JS: var -> const

File: README.md [hunk 2]
```diff
@@ -10,9 +10,8 @@ You can test it here - http://thetax.nl
 
 </p>
 
-More infromation about tax brackets [here](http://www.belastingdienst.nl/wps/wcm/connect/bldcontentnl/belastingdienst/prive/inkomstenbelasting/heffingskortingen_boxen_tarieven/boxen_en_tarieven/overzicht_tarieven_en_schijven/u-hebt-in-2017-de-aow-leeftijd-nog-niet-bereikt) (for 2017)
-and [here](http://www.belastingdienst.nl/wps/wcm/connect/bldcontentnl/belastingdienst/prive/inkomstenbelasting/heffingskortingen_boxen_tarieven/boxen_en_tarieven/overzicht_tarieven_en_schijven/u_hebt_in_2016_de_aow_leeftijd_nog_niet_bereikt) (for 2016) 
-and [here](http://www.belastingdienst.nl/wps/wcm/connect/bldcontentnl/belastingdienst/prive/inkomstenbelasting/heffingskortingen_boxen_tarieven/boxen_en_tarieven/overzicht_tarieven_en_schijven/u_hebt_in_2015_de_aow_leeftijd_nog_niet_bereikt) (for 2015).
+More information about tax brackets [here](https://www.belastingdienst.nl/bibliotheek/handboeken/html/boeken/HL/index.html) (for current)
+and [here](https://www.belastingdienst.nl/wps/wcm/connect/nl/zoeken/zoeken?q=Handboek+Loonheffingen) (from 2015 until current year).
```
Comment (stevermeister) [inline README.md:14]: 👍

File: src/calc/calc.component.js [hunk 3]
```diff
@@ -253,39 +262,84 @@ let calcComponent = {
...
+     * Social Security Contribution (Volksverzekeringen) Component of Tax Credit
+     * https://www.belastingdienst.nl/wps/wcm/connect/bldcontentnl/belastingdienst/prive/werk_en_inkomen/sociale_verzekeringen/premies_volks_en_werknemersverzekeringen/volksverzekeringen/hoeveel_moet_u_betalen
+     * 
+     * @param {string} year Year to retrieve information from
+     * @param {boolean} older Whether is after retirement age or not
+     * @param {boolean} socialSecurity Whether social security will be considered or not
+     * @param {boolean} [print] Print output table to console log or not
+     * @returns {number} Social Security contribution percentage to apply to wage credit
+     */
+    function getSocialCredit(year, older, socialSecurity, print = isDebug) {
```
Comment (stevermeister) [inline src/calc/calc.component.js]: `print` - I would rather not mix calculations and output logic here.

File: src/calc/calc.component.js [hunk 4]
```diff
@@ -299,39 +353,106 @@ let calcComponent = {
...
+     * https://www.belastingdienst.nl/wps/wcm/connect/bldcontentnl/themaoverstijgend/brochures_en_publicaties/nieuwsbrief-loonheffingen-2020
+     * 
+     * @param {object[]} brackets Rate brackets to extract information from
+     * @param {number} salary Taxable wage that will be used for calculation
+     * @param {string} kind Property name to be extracted from bracket
+     * @param {number} [multiplier] Scalar value to multiple against final result
+     * @param {boolean} [print] Print output table to console log or not
+     * @returns {number} Accumulated tax/credit amount to be used to calculate the net income
+     */
+    function getRates(brackets, salary, kind, multiplier = 1, print = isDebug) {
```
Comment (stevermeister) [inline src/calc/calc.component.js]: same here

File: src/calc/calc.component.js (hunk 3 above)
Comment (eduardomourar) [reply src/calc/calc.component.js]: What do you suggest? the print output has be done during the calculation steps, so just return an object like this always:
```json
{
"calculation": ...,
"printOutput": ...,
}
```

Comment (stevermeister) [reply src/calc/calc.component.js]: Could I ask you to split PR into 2: 
- nice fixes - that I will merge right away
- print output - to discuss it a bit more

thank you!

Comment (eduardomourar) [reply src/calc/calc.component.js]: Done. Removed print output from this PR, and I will create another as soon as this one is merged.

Comment (stevermeister) [reply src/calc/calc.component.js]: great, thank you. merging...

**Pull request merged**